config/routes.rb

```diff
@@ -9,6 +9,7 @@ def add_swagger_route(http_method, path, opts = {})
 
     full_path = path.gsub(/{(.*?)}/, ':\1')
     scope :as => :api, :module => "api", :path => prefix do
+      match "/v0/*path", :via => [:delete, :get, :options, :patch, :post], :to => redirect(:path => "/#{prefix}/v0.0/%{path}", :only_path => true)
```

Comment:
I think the idea was to redirecting to the latest minor version, redirecting to `0.0` was meant to be a temporary change.  cc @syncrou

Reply:
Correct, we will be defaulting to v0.1.0 on service_portal after the resource routes are merged ( as a separate PR. )

Reply:
Ok, `v0.0` is the only available version so far in approval service. Will update to `v0.1.0` when it's needed.

Reply:
Isn't it `0.0.1` with last PR(#47) merged?

Reply:
@bzwei `0.0.1` is the version for openapi.json. I think we only care the major and minor versions so far.